android/engine/src/test/java/org/smartregister/fhircore/engine/task/FhirCarePlanGeneratorTest.kt

```diff
@@ -1604,17 +1642,15 @@ class FhirCarePlanGeneratorTest : RobolectricTest() {
   }
 
   @Test
-  fun `updateDependentTaskDueDate should have no dependent tasks`() = runBlocking {
+  fun `updateDependentTaskDueDate should have no dependent tasks`() {
     coEvery { fhirEngine.search<Task>(any<Search>()) } returns emptyList()
-
     opv0.apply {
       status = TaskStatus.REQUESTED
       partOf = emptyList()
     }
-
-    val updatedTask = opv0.updateDependentTaskDueDate(defaultRepository, fhirEngine)
-
-    coVerify { fhirEngine.search<Task>(any<Search>()) }
```

Comment:
we should either keep the coVerify, or remove it and the coEvery on line 1646 -- same applies throughout

Reply:
Alright